cfme/tests/cloud/test_provisioning.py

```diff
@@ -206,9 +206,17 @@ def test_provision_from_template_with_attached_disks(
         for i, volume in enumerate(volumes):
             device_mapping.append((volume, DEVICE_NAME.format(chr(ord("b") + i))))
         # Set up automate
+
+        ocls = automate.Class(name="Methods",
+            namespace=automate.Namespace.make_path("Cloud", "VM", "Provisioning", "StateMachines",
+            parent=automate.Domain(name="ManageIQ (Locked)")))
+
         method = automate.Method(
             name="openstack_PreProvision",
-            cls=cls)
+            cls=ocls)
+
+        method.copy_to(domain)
```

Comment:
`method = method.copy_to(domain)`? Just curious if this has any effect in this place. But I believe it should be like that either way since it can mislead people who don't know how this would exactly work.